src/Root.tsx

```diff
@@ -0,0 +1,19 @@
+import { HashRouter, Navigate, Route, Routes } from 'react-router-dom';
+import { HomePage } from './components/HomePage';
+import { PeoplePage } from './components/PeoplePage';
+import { App } from './App';
+
+export const Root = () => (
+  <HashRouter>
+    <Routes>
+      <Route path="/" element={<App />}>
+        <Route index element={<HomePage />} />
+        <Route path="/home" element={<Navigate to="/" replace />} />
+        <Route path="/people" element={<PeoplePage />}>
+          <Route path="/people/:slug" element={<PeoplePage />} />
```

Comment:
The nested route for `/people/:slug` uses the same component `PeoplePage` as the parent route `/people`. If `PeoplePage` is intended to render different content based on the `slug` parameter, ensure that it handles this parameter correctly. Otherwise, consider using a different component for the nested route if the content is supposed to differ.